var/spack/repos/builtin/packages/unison/package.py

```diff
@@ -20,19 +20,12 @@ class Unison(Package):
 
     license("GPL-3.0-or-later")
 
-    version("2.51.2", sha256="a2efcbeab651be6df69cc9b253011a07955ecb91fb407a219719451197849d5e")
-    version("2.48.15v4", sha256="f8c7e982634bbe1ed6510fe5b36b6c5c55c06caefddafdd9edc08812305fdeec")
+    version("2.53.3", sha256="aaea04fc5bc76dcfe8627683c9659ee4c194d4f992cc8aaa15bbb2820fc8de46")
```

Comment:
Versions are typically deprecated for a release before being removed.  See the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions.

Reply:
I added the older versions back in, but I was unable to test this.
On develop, OCaml < 4.13.1 does not build for me.
And with OCaml 4.13.1, the older unison versions don't build.

I think the older versions are broken in develop without further patching to make them 4.13.1 compatible.